### PR TITLE
docs: update config guide

### PR DESCRIPTION
**Pull request by tamazlykar**
Issue #1167

### CODE REVIEW

File: config-guide.md [hunk 1]
```diff
@@ -1,201 +1,270 @@
...
+
+### Allow Reordering Sidebar
 A boolean value which allows or forbids a user to reorder links in the main sidebar. 
 ```
  "allowReorderingSidebar": false
 ```
 
-### Disk Offering Parameters
-This configuration allows a user to set parameters of Disk Offerings that will be shown in Disk Offerings Table.
+### Configure Sidenav
```
Comment (ershovaelena) [inline config-guide.md]: "Configure left-side menu"

File: config-guide.md [hunk 2]
```diff
@@ -1,201 +1,270 @@
...
+#Config Guide
 
-### Default domain URL
+See the [config-example.json](https://github.com/bwsw/cloudstack-ui/blob/master/src/config/config-example.json) of a config.json
 
-Domain URL used to fill the 'Domain' field in the login form
+## General
 
-    "defaultDomain": "domain"
+### Default domain
```
Comment (ershovaelena) [inline config-guide.md]: **D**omain

File: config-guide.md [hunk 3]
```diff
@@ -1,201 +1,270 @@
...
+```
+"extensions": {
+  "webShell": true,
+  "pulse": false
+}
+```
+
+Please check [Wiki](https://github.com/bwsw/cloudstack-ui/wiki/Plugins) for full extension configuration.
+
+## Virtual machines settings
```
Comment (ershovaelena) [inline config-guide.md]: **M**achines **S**ettings

File: config-guide.md [hunk 4]
```diff
@@ -1,201 +1,270 @@
...
    * ruleid: a unique identifier
    * protocol: either 'tcp', 'udp' or 'icmp'
    * cidr: subnet mask (e.g. 0.0.0.0/0)
    * For TCP and UDP: startport and endport
    * For ICMP: icmpcode and icmptype
 
-### VM Colors
 
-The set of colors for virtual machines in hexadecimal format. You can specify any colors you like.
+## Images settings
```
Comment (ershovaelena) [inline config-guide.md]: **S**ettings

File: config-guide.md [hunk 5]
```diff
@@ -1,201 +1,270 @@
...
-```
-"blue-red"
-"indigo-pink"
-```
-Is not specified, blue-red one is used.
-```
- "defaultThemeName": "blue-red"
-```
-### Offering Availability
+## User app settings
```
Comment (ershovaelena) [inline config-guide.md]: **A**pp **S**ettings

File: config-guide.md [hunk 6]
```diff
@@ -244,36 +262,152 @@ For example (default values),
...
+]
 ```
 
-### Default Time Format
-Allows you to predefine the setting of the time format. Possible values:
-- "auto" - value depends on the interface language (default)
-- "hour12" - 12-hour time
-- "hour24" - 24-hour time
+### Offering Compatibility Policy
+This configuration allows to restrict compute offering change based on the compute offering host tags.
```
Comment (MaslovaEV) [inline config-guide.md]: This configuration allows **you** to restrict

**Pull request merged**